debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -229,25 +230,34 @@
 
     protected List<SchemaAndValueField> schemasAndValuesForArrayTypes() {
        return Arrays.asList(new SchemaAndValueField("int_array", SchemaBuilder.array(Schema.OPTIONAL_INT32_SCHEMA).optional().build(),
-                                Arrays.asList(1, 2, 3)),
+                                expectedArray(1, 2, 3)),
                             new SchemaAndValueField("bigint_array", SchemaBuilder.array(Schema.OPTIONAL_INT64_SCHEMA).optional().build(),
-                                Arrays.asList(1550166368505037572L)),
+                                expectedArray(1550166368505037572L)),
                             new SchemaAndValueField("text_array", SchemaBuilder.array(Schema.OPTIONAL_STRING_SCHEMA).optional().build(),
-                                Arrays.asList("one", "two", "three")),
+                                expectedArray("one", "two", "three")),
                             new SchemaAndValueField("char_array", SchemaBuilder.array(Schema.OPTIONAL_STRING_SCHEMA).optional().build(),
-                                    Arrays.asList("cone      ", "ctwo      ", "cthree    ")),
+                                expectedArray("cone      ", "ctwo      ", "cthree    ")),
                             new SchemaAndValueField("varchar_array", SchemaBuilder.array(Schema.OPTIONAL_STRING_SCHEMA).optional().build(),
-                                    Arrays.asList("vcone", "vctwo", "vcthree"))
+                                expectedArray("vcone", "vctwo", "vcthree")),
+                            new SchemaAndValueField("date_array", SchemaBuilder.array(Date.builder().optional().schema()).optional().build(),
+                                expectedArray(17109, 17110, 17111))
```

Comment:
How about specifying those via `LocalDate.of(2016, NOVEMBER, 4).toEpochDay()`. Makes it a bit easier to reason about this test.